drafts/20003.adoc

```diff
@@ -0,0 +1,459 @@
+---
+layout: post
+title: "New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tomjenningss
+author_github: https://github.com/tomjenningss
+seo-title: New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3
+seo-description: Yu can use the Kafka client API. Allowing users to access Kafka specific message properties such as `message key` and `message headers`, not just the payload of the specific message. Also, you can now add the SameSite Cookie on the session, with three values on the SameSite attribute.
+---
+
+= New Kafka Client API allowing Kafka specific message properties in Open Liberty 20.0.0.3
+
+Tom Jennings <https://github.com/tomjenningss>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+// tag::intro[]
+In Open Liberty 20.0.0.3, you can use the Kafka client API. Allowing users to access Kafka specific message properties such as `message key` and `message headers`, not just the payload of the specific message as you previously had with MicroProfile Reactive `Message` API. Also, you can now add the SameSite Cookie on the session, with three values on the SameSite attribute.
```

Comment:
Can you tweak the paragraph so that it reads:

"In Open Liberty 20.0.0.3, you can now access Kafka specific properties such as the message key and message headers, not just the payload of the message, as was the case with the basic MicroProfile Reactive Messaging `Message` API. Also, you can now set the SameSite attribute on the Session cookie, the LTPA, and JWT cookies as well as application-defined cookies."

(I just copied the wording for the SameSite attribute from the SameSite section to make sure it was accurate, as I don't think the edits you made are quite what it was saying. It's still short enough so it's fine like this.)